src/continuous-scroll.js

```diff
@@ -0,0 +1,202 @@
+/* global TweenLite, TimelineLite, Linear */
+import React from "react";
+import "gsap";
+
+let tween = null,
+  timeline = null;
+
+const ContinuousScroll = React.createClass({
+  propTypes: {
+    type: React.PropTypes.oneOf(["none", "continuous", "page"]),
+    speed: React.PropTypes.oneOf(["slow", "medium", "fast"]),
+    pause: React.PropTypes.number,
+    height: React.PropTypes.number.isRequired,
+    data: React.PropTypes.array.isRequired,
+    isRefreshing: React.PropTypes.bool,
```

Comment:
@stulees If the data has changed, yes. For Spreadsheet Widget the data is all we currently store in the state so any change to that will automatically re-render and call that function. But if we start to store other things in the state, then changing some other piece of it would still execute this code unnecessarily. Hmmm...so now I'm back to having it as a prop that would be set to false in the Widget any time a piece of state is updated that is not the data. Any other ideas?